adb-vagrantfile-tests/k8s-tests/run.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+
+import json
+import urllib
+import subprocess
+import sys
+import os
+
+
+url_base = "http://admin.ci.centos.org:8080"
+api_key = os.environ['API_KEY']
+count = os.environ['MACHINE_COUNT']
```

Comment:
If you don't want to get keyerror in case 'API_KEY' not present then better use `os.getenv`
```
>>> os.getenv('HOME')
'/home/prkumar'
>>> os.getenv('HOM')
>>> os.environ['HOME']
'/home/prkumar'
>>> os.environ['HOM']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/UserDict.py", line 23, in __getitem__
    raise KeyError(key)
KeyError: 'HOM'
```